src/routes/tasks.rs

```diff
@@ -93,3 +93,16 @@ pub async fn update(
     let task = update.build(task)?.update(&db).await?;
     Ok(Json(task))
 }
+
+pub mod collector_auth_tokens {
+    use super::*;
+    pub async fn index(
+        _: &mut Conn,
+        (task, client): (Task, AggregatorClient),
+    ) -> Result<impl Handler, Error> {
+        //        let leader = task.leader_aggregator(&db).await?;
+        //        let client = leader.client(client);
```

Comment:
Commented out lines should be removed

Reply:
In this case they're the lines we'll need post #216. I'll stash locally and remove